lib/rubocop/cop/lint/void.rb

```diff
@@ -143,6 +143,15 @@ def in_void_context?(node)
 
           return false unless parent && parent.children.last == node
 
+          value_returning_methods = %i[
```

Comment:
Seems a bit weird to inline this array in the method. Probably it should be a constant instead.